### PR TITLE
Ide probe ignore errors by parts of stack trace

### PR DESCRIPTION
**Pull request by LukaszKontowski**
Closes https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1063

### CODE REVIEW

**Review by PawelLipski — APPROVED** [review]

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note
 [versions]
 checker = "3.24.0"
-ideProbe = "0.42.0"
+ideProbe = "0.41.0+30-f11e897b-SNAPSHOT"
```
Comment (LukaszKontowski) [inline gradle/libs.versions.toml:6]: To be changed to `0.43.0` when released - if the `0.43.0` release will have the https://github.com/VirtusLab/ide-probe/pull/263 changes included

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note
 [versions]
 checker = "3.24.0"
-ideProbe = "0.41.0+30-f11e897b-SNAPSHOT"
+ideProbe = "0.0.0+532-f11e897b-SNAPSHOT"
```
Comment (PawelLipski) [inline gradle/libs.versions.toml:6]: Note that your fork doesn't have tags: https://github.com/LukaszKontowski/ide-probe

image

Probably once the tags are pushed to your fork, this version can be unified (the same version used both for local builds and CI builds)

Comment (LukaszKontowski) [reply gradle/libs.versions.toml:6]: Sure, I will fix it

Comment (LukaszKontowski) [reply gradle/libs.versions.toml:6]: fixed!

**Review by PawelLipski — APPROVED** [review]
:fantastycznie:

**Pull request merged**